cf-net/cf-net.c

```diff
@@ -129,6 +131,7 @@ static const char *const HINTS[] =
     "Minimum TLS version to use",
     "TLS ciphers to use (comma-separated list)",
     "Specify CFEngine protocol to use. Possible values: 'classic', 'tls', 'cookie', 'filestream', 'latest' (default)",
+    "Print RSYNC performance statistics",
```

Comment:
Out of curiosity, why all caps? I don't see that on their website for example.

https://rsync.samba.org/

Reply:
You're right :+1: